spec/requests/articles/articles_spec.rb

```diff
@@ -301,13 +301,7 @@
       expect { get "#{article.path}/stats" }.to raise_error(Pundit::NotAuthorizedError)
     end
 
-    it "returns unauthorized if the user is not pro" do
-      article = create(:article, user: user)
-      expect { get "#{article.path}/stats" }.to raise_error(Pundit::NotAuthorizedError)
-    end
```

Comment:
This test no longer applies since we are no longer authorizing against the `:pro` role at all for the `/stats` route of an article, so I removed it!